.github/workflows/main.yml

```diff
@@ -149,6 +149,7 @@ jobs:
           mkdir -p .tests/data
           curl -L https://github.com/thomasbtf/small-kraken-db/raw/master/B.1.1.7.reads.1.fastq.gz > .tests/data/B117.1.fastq.gz
           curl -L https://github.com/thomasbtf/small-kraken-db/raw/master/B.1.1.7.reads.1.fastq.gz > .tests/data/B117.2.fastq.gz
+          curl -L https://github.com/thomasbtf/small-kraken-db/raw/master/ont_reads.fastq.gz > .tests/data/ont_reads.fastq.gz
```

Comment:
Wouldn't it be better to move those files to the official repo?

Reply:
I wanted to avoid making this repo too big, space-wise. These files would be forever in the history of the repo, thus in each new "master-merge".

I would like to suggest this as a long-term solution: https://github.com/marketplace/actions/qc-benchmark-action

However, this is another big PR itself.